### PR TITLE
fix(main menu) share case as link with id when sharing via cases menu

### PR DESCRIPTION
**Pull request by yuvalbl**
should solve issue #937

### CODE REVIEW

File: src/app/app/app-effects/effects/status-bar.app.effects.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export class StatusBarAppEffects {
 			return state.selectedCase.id;
 		})
 		.map((caseId: string) => {
-			return new CopyCaseLinkAction(caseId);
+			return new CopyCaseLinkAction({caseId: caseId, shareCaseAsQueryParams: true});
```
Comment (AmitMY) [inline src/app/app/app-effects/effects/status-bar.app.effects.ts]: image

File: src/app/packages/menu-items/cases/actions/cases.actions.ts [hunk 2]
```diff
@@ -188,6 +188,6 @@ export class SaveCaseAsSuccessAction implements Action {
 export class CopyCaseLinkAction implements Action {
 	type = CasesActionTypes.COPY_CASE_LINK;
 
-	constructor(public payload: string) {
+	constructor(public payload: {caseId: string, shareCaseAsQueryParams?: boolean}) {
```
Comment (AmitMY) [inline src/app/packages/menu-items/cases/actions/cases.actions.ts]: image

File: src/app/packages/menu-items/cases/components/cases-table/cases-table.component.ts [hunk 3]
```diff
@@ -111,7 +111,7 @@ export class CasesTableComponent implements OnInit {
 	}
 
 	shareCase(caseId: string) {
-		this.store$.dispatch(new CopyCaseLinkAction(caseId));
+		this.store$.dispatch(new CopyCaseLinkAction({caseId: caseId}));
```
Comment (AmitMY) [inline src/app/packages/menu-items/cases/components/cases-table/cases-table.component.ts]: image

File: src/app/app/app-effects/effects/status-bar.app.effects.ts (hunk 1 above)
Comment (yuvalbl) [reply src/app/app/app-effects/effects/status-bar.app.effects.ts]: image

File: src/app/packages/menu-items/cases/actions/cases.actions.ts (hunk 2 above)
Comment (yuvalbl) [reply src/app/packages/menu-items/cases/actions/cases.actions.ts]: image

File: src/app/packages/menu-items/cases/components/cases-table/cases-table.component.ts (hunk 3 above)
Comment (yuvalbl) [reply src/app/packages/menu-items/cases/components/cases-table/cases-table.component.ts]: image

**Pull request merged**